lib/apiml.js

```diff
@@ -202,6 +203,93 @@ ApimlConnector.prototype = {
     });
   },
   
+  getInstanceId() {
+    return this.zluxServerEurekaClient.config.instance.instanceId;
+  },
+  
+  getZluxInstances() {
+    const eurekaClient = this.zluxServerEurekaClient;
+    return new Promise((resolve, reject) => {
```

Comment:
because `registryFetchInterval=30000`, does this really wait up to 30 seconds to resolve? is that ok for how the code works? i will read more below.

Reply:
The Eureka client will fetch Eureka registry every 30 seconds so we need to wait less than 30 seconds in 99.9% of cases.